interview/password.py

```diff
@@ -15,5 +15,26 @@ def generate_password() -> str:
     # https://docs.python.org/3/library/random.html#random.randint
 
     # TODO implement generate_password function
+    import random 
+    password = ''
     
-    return ''
\ No newline at end of file
+    alphabet = 'abcdefghijklmnopqrstuvwxyz'
+    r = alphabet[random.randint(0, len(alphabet)-1)]
```

Comment:
this is fine, but when you use random this way you end up with passwords that have matching first letters (ex: 'aaaaa' ...)